app/jobs/process_patient_changesets_job.rb

```diff
@@ -22,6 +22,8 @@ def perform(patient_changeset, step_name = nil)
         nhs_number: pds_patient&.nhs_number
       }
 
+      patient_changeset.save!
+
```

Comment:
Can we have this as an alternative (i.e. `else` section) to the `if` statement below? Just to prevent extra DB hits.